modules/account-lib/src/coin/cspr/transactionBuilder.ts

```diff
@@ -248,6 +248,16 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
       if (_sourceKeyPair.getKeys().prv === key.key.toUpperCase()) {
         throw new SigningError('Repeated sign: ' + key.key);
       }
+      // Try to get extended keys in order to validate them
+      let xprv;
+      try {
+        xprv = _sourceKeyPair.getExtendedKeys().xprv;
+      } catch (err) {
+        return;
```

Comment:
Why we don't log the error?

Reply:
Here the return is to break the for each in the line 248, we don't want exception thrown. Whit this we want to avoid an unexpected behavior when the key pair doesn't have extended keys, if the extended keys exists, they will be validate correctly

Reply:
:rocket: gotcha! thanks.